test/runner.zig

```diff
@@ -208,33 +217,25 @@ fn mainTerminal() void {
 		testing.log_level = .warn;
 
 		const test_node = root_node.start(test_fn.name, 0);
-		if(!have_tty) {
-			std.debug.print("{d}/{d} {s}...", .{i + 1, test_fn_list.len, test_fn.name});
-		}
+
+		const padding_length: usize = max_length - test_fn.name.len + 3;
+		const padding = padding_buffer[0..padding_length];
+
+		std.debug.print("{d: >4}/{d: <4} {s}{s}", .{i + 1, test_fn_list.len, test_fn.name, padding});
 		is_fuzz_test = false;
 		if(test_fn.func()) |_| {
 			ok_count += 1;
 			test_node.end();
-			if(!have_tty) std.debug.print("OK\n", .{});
+			std.debug.print("OK\n", .{});
```

Comment:
If we are touching these lines anyways, have you thought about adding colors, green for OK, yellow for SKIP and red for FAIL

Reply:
Aren't color codes shell dependent?